lxd/storage/drivers/driver_ceph.go

```diff
@@ -405,6 +405,23 @@ func (d *ceph) Validate(config map[string]string) error {
 
 // Update applies any driver changes required from a configuration change.
 func (d *ceph) Update(changedConfig map[string]string) error {
+	newSize := changedConfig["ceph.osd.pool_size"]
+	if newSize != d.config["ceph.osd.pool_size"] && newSize != "" {
```

Comment:
Is this `newSize != d.config["ceph.osd.pool_size"]` redundant, if they were equal it wouldnt be in changedConfig right?

Reply:
Perhaps clearer to test for `newSize, changed := changedConfig["ceph.osd.pool_size"]` instead

Reply:
Ah good point, thanks!